lars-reimann/api-editor
# refactor: use usage counts in generate command

Closes #448.
Closes #481.

### Summary of Changes

* Implement `UsageCountStore` class.
* Use `UsageCountStore` class and corresponding JSON files as input to `generate` command.
* Update test data accordingly.
* Split large test data file into multiple files, one for each kind of annotation.

### Testing Instructions

1. Open a terminal in the `package-parser` directory.
2. Run `parse-package generate -a ..\api-editor\data\api\scikit-learn__sklearn__1.0.2__api.json -u ..\api-editor\data\usages\scikit-learn__sklearn__0.24.2__usage_counts.json -o out\out.json`.
3. Check the created annotations file `out/out.json`.
